### DISCUSSION TITLE
Livewire mount method using Volt class based API prevents proper error page

### DISCUSSION THREAD

**Discussion by ChrisReid** [livewire/livewire#8687] [7. Bugs (archived)]
### Livewire version

v3.5.4

### Laravel version

v11.17.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Install repository
2. Navigate to /counter
3. Notice the Symphony error page showing: htmlspecialchars(): Argument #1 ($string) must be of type string, Closure given
4. Comment out or delete the mount method
5. Notice the proper Laravel error page

### Repository or Wirebox URL

https://github.com/cloudest-co/laravel-livewire-bug-report

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by jsandfordhughes** [comment]
Just having this issue too however it seems like it is linked to Volt specifically. I cannot seem to replicate it while using a standard class based Livewire component.
